sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/TypesafeBackendImpl.kt

```diff
@@ -27,6 +27,8 @@ internal class TypesafeBackendImpl(private val backend: Backend) : TypesafeBacke
     override val network: ZcashNetwork
         get() = ZcashNetwork.from(backend.networkId)
 
+    override suspend fun getAccounts(): List<Account> = backend.getAccounts().map { Account(it.accountId.toInt()) }
```

Comment:
It turns out that this is a bug. I had forgotten that `Account` is used universally though the Android SDK to specifically mean the ZIP 32 account index, and we work around this fact in the Rust backend by looking for an account with a matching ZIP 32 account index. I will fix this in a follow-up PR.